### DISCUSSION TITLE
Is there Kogito ext for Quarkus 3?

### DISCUSSION THREAD

**Discussion by faermanj** [quarkusio/quarkus#41343] [Community]
It seems there is only a quarkus 2 version of the kogito extension, is there any story / plan here?
https://quarkus.io/version/2.13/guides/kogito

**Comment by fah-kmd-dk** [comment]
hi @faermanj

better late than ever - check this kogito discussion : https://github.com/apache/incubator-kie-issues/discussions/1976 

Br Frank
